src/app.jl

```diff
@@ -108,29 +136,86 @@ end
 """
 
 function dash(name::String;
```

Comment:
@rpkyle I will soon remove the version with `layout_maker`, so that it does not confuse you. Now this will break up the old tests, which I am slowly rewriting with the new functionality. Just ignore this option for now